src/os/exec_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package os_test
+
+import (
+	. "os"
+	"runtime"
+	"testing"
+)
+
+func TestFindProcess(t *testing.T) {
+	// NOTE: For now, we only test the Linux case since only exec_posix.go is currently the only implementation.
+	if runtime.GOOS == "linux" {
```

Comment:
exec_posix.go does in fact include Windows, and because it's all no-ops anyway I think it would be fine to test this on any platform (including Windows).